packages/expo-symbols/.gitignore

```diff
@@ -0,0 +1,57 @@
+# OSX
```

Comment:
nit: though this may coming from expo-module-template. we may like to leverage **expo/expo/.gitignore** and keep only necessary items here and easier for maintenance.